yt.gemspec

```diff
@@ -1,4 +1,3 @@
-# coding: utf-8
```

Comment:
what is the difference?

Reply:
It's necessary for old Ruby (< 2.0 IIRC), which we should drop support for.

Reply:
thank you!